src/NLog/LogFactory.cs

```diff
@@ -187,7 +187,7 @@ public LoggingConfiguration Configuration
                         }
                         catch (Exception exception)
                         {
-                            InternalLogger.Warn("Cannot start file watching: {0}. File watching is disabled", exception);
+                            InternalLogger.Warn(exception, "Cannot start file watching. File watching is disabled");
```

Comment:
Ok. but what about `MustBeRethrownImmediately()`. If we don't rethrow those exceptions, bad things might happen.